src/workerd/api/tests/global-scope-test.js

```diff
@@ -134,7 +134,7 @@ export const queueMicrotask = {
       throws(() => globalThis.queueMicrotask(i), {
         message:
           "Failed to execute 'queueMicrotask' on 'ServiceWorkerGlobalScope': " +
-          "parameter 1 is not of type 'Function'.",
+          "parameter 1 is not of type 'function'.",
```

Comment:
This appears to just be an artifact of the type mapping system that uses `Function` for v8::Function and `function` for `jsg::Function`. Ah well.